contrib/terraform/openstack/modules/compute/outputs.tf

```diff
@@ -1,3 +1,11 @@
 output "k8s_master_ips" {
   value = concat(openstack_compute_instance_v2.k8s_master_no_floating_ip.*, openstack_compute_instance_v2.k8s_master_no_floating_ip_no_etcd.*)
 }
+
+output "k8s_masters_reserved_ips" {
```

Comment:
Wouldn't it be possible to have this in the `k8s_master_fips` variable in "outputs.tf" in the "ips" module instead?

I'm just thinking that then any script or such that assumes the FIP is there would still work even with reserved IPs

But if you think that is too much trouble then I'm fine leaving it here

Reply:
I think I looked at doing it that way but encountered some issues, but I can try looking at it again. I agree it would be nicer

Reply:
I am not sure how I would go about it, since the output for `openstack_networking_floatingip_v2.k8s_masters` is a map with a bunch of [attributes](https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs/resources/networking_floatingip_v2#argument-reference), I am not sure how to combine it with the reserved_ips. I tried by creating a local object for `reserved_ips` that have the `address` attribute, but then it complains that not all map elements/attributes are the same.

Merging maps for both reserved and created floating IPs with only the `address` attribute could be a solution, see https://github.com/elastisys/kubespray/pull/19/commits/2b80ad636dbbc83d0fb46526d8ab88bbb4061bbc for how this could look like, in this case only for `k8s_masters`. If this solution suffices I will update it for `k8s_nodes` as well.

Reply:
I think that looks like a good solution. As long as we can assure that stuff other than the address is used anywhere so we don't break anything.